app/controllers/spree/admin/payment_methods_controller.rb

```diff
@@ -80,7 +81,7 @@ def validate_payment_method_provider
       end
 
       def payment_method_params
-        params.require(:payment_method).permit!
+        params.require(:payment_method).permit(:type, :name, :description, :display_on, :auto_capture, :active, store_ids: [])
```

Comment:
@aleksandarpetrushev there are several other preferences for different payment methods, this will break that code

Reply:
@damianlegawiec Do we have a list of all of them, or should I revert this change?

Reply:
@aleksandarpetrushev we don't have a full list, it depends on the payment provider, there are many custom ones with custom preferences

Reply:
@damianlegawiec I checked a few providers, and these params were the same (didn't check all of them). Could there be a chance that these custom preferences are part of https://github.com/spree/spree_backend/blob/50b044b4b27d677ebc4eadd022160fbacd4b6835/app/controllers/spree/admin/payment_methods_controller.rb#L87 and used when updating the payment method after creation?

Reply:
@aleksandarpetrushev to be frank, I'm not sure, you need to test, the current solution will not work so this isn't ready for review yet

Reply:
@damianlegawiec tested with all providers, the payment_method_params are the same. What differs is AFTER you create the payment method, based on the provider you have additional settings, which are whitelisted in preferences_params.